spiechu/symfony-commons-bundle/0.1/config/packages/spiechu_symfony_commons.yaml

```diff
@@ -0,0 +1,24 @@
+spiechu_symfony_commons:
+    get_method_override:
+        enabled:              false
+        #listener_service_id:  spiechu_symfony_commons.event_listener.get_method_override_listener
```

Comment:
Do we really need these comments?
Will they help the user or would it be better to refer them to the readme?

Reply:
To be honest I was suggested by other already accepted recipes.
I saw configuration dumps fully commented out. I'm not sure what is better for DX.

Of course I can remove any commented out configuration and point user to readme.

Reply:
Hm, I see. 

The rule of thumb is: Do you think most (80%) of the users will change these values? If not, we should refer them to your docs instead.